index.js

```diff
@@ -104,27 +140,36 @@ function myMap(array, callback) {
   return result
 }
 
+
 // ***** Callbacks - Question 1 *****
 
+function triple (number) {
+  console.log(number * 3)
```

Comment:
This function should have a `return` instead of a `console.log` - the purpose of the `myMap` function is to create a new array with each element based on the callback function, so that callback needs to return a value to be added to the new array. With this code, we'd get back an array of `[undefined, undefined, undefined...]` instead of `[3,6,9...]`

```js
function triple (number) {
  return number * 3
}
```